src/Miunie.ConsoleApp/ConsoleMenu.cs

```diff
@@ -138,7 +138,13 @@ private void PrintToBottom(string message)
         {
             var prevCursorLeft = Console.CursorLeft;
             var prevCursorTop = Console.CursorTop;
-            Console.SetCursorPosition(0, Console.WindowHeight - 4);
```

Comment:
This code would make the app crash if the window was small so that the CursorPosition became negative.
Fixed so that it doesn't crash.